code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,360 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+
+/obj/machinery/clonepod/traitorous
+	name = "experimental cloning pod traitorous"
+	desc = "An ancient cloning pod. It seems to be an early prototype of the experimental cloners used in Nanotrasen Stations."
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_0"
+	req_access = null
+	circuit = /obj/item/circuitboard/machine/clonepod/traitorous
+	internal_radio = FALSE
+	var/emmaged = FALSE
+
+//Start growing a human clone in the pod!
+/obj/machinery/clonepod/traitorous/growclone(clonename, ui, mutation_index, mindref, last_death, datum/species/mrace, list/features, factions, list/quirks, datum/bank_account/insurance)
+	if(panel_open)
+		return NONE
+	if(mess || attempting)
+		return NONE
+
+	attempting = TRUE //One at a time!!
+	countdown.start()
+
+	var/mob/living/carbon/human/H = new /mob/living/carbon/human(src)
+
+	/*var/expl_text stripped_input(admin, "(Firstidk?)Custom objective:", "ObjectiveSecond", explanation_text)
+	var/datum/objective/custom/O
+		name = "custom"
+		explanation_text = expl_text
+
+	*/
+
+	H.hardset_dna(ui, mutation_index, H.real_name, null, mrace, features)
+
+	if(efficiency > 2)
+		var/list/unclean_mutations = (GLOB.not_good_mutations|GLOB.bad_mutations)
+		H.dna.remove_mutation_group(unclean_mutations)
+	if(efficiency > 5 && prob(20))
+		H.easy_randmut(POSITIVE)
+	if(efficiency < 3 && prob(50))
+		var/mob/M = H.easy_randmut(NEGATIVE+MINOR_NEGATIVE)
```

Comment:
easy_randmut returns TRUE or FALSE so var/mob/M will always be null making ismob(M) pointless

Reply:
It has been in the original experimental cloner, so I haven't really checked that one.